### PR TITLE
add the next.js handbook translated by abdelhadi

### PR DESCRIPTION
**Pull request by AhmedElTabarani**
## What does this PR do?
Add resource(s) | Remove resource(s) | Add info | Improve repo

## Checklist:
- [ ] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md)
- [ ] Search for duplicates.
- [ ] Include author(s) and platform where appropriate.
- [ ] Put lists in alphabetical order, correct spacing.
- [ ] Add needed indications (PDF, access notes, under construction)

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by AhmedElTabarani** [comment]
This PR for No. 2 of this #6764

**Comment by LuigiImVector** [comment]
Being that it is a single resource and Next.js is a framework of JavaScript I propose to avoid to create a new category and to insert it on JavaScript

**Comment by AhmedElTabarani** [comment]
> Being that it is a single resource and Next.js is a framework of JavaScript I propose to avoid to create a new category and to insert it on JavaScript

@eshellman said here https://github.com/EbookFoundation/free-programming-books/issues/6764#issuecomment-1064174691

> 3. definite yes on the Next.js book

File: books/free-programming-books-ar.md [hunk 1]
```diff
@@ -8,6 +8,7 @@
 * [HTML/CSS](#htmlcss)
 * [Introduction to Programming in Arabic](#introduction-to-programming-in-arabic)
 * [JavaScript](#javascript)
+  * [Next.js](#next.js)
```
Comment (davorpa) [inline books/free-programming-books-ar.md:11]:
```suggestion
* [JavaScript](#javascript)
```

File: books/free-programming-books-ar.md [hunk 2]
```diff
@@ -58,6 +59,11 @@
 * [Eloquent Javascript (Second Edition)&rlm;](https://rabahboudia.gitbooks.io/arabic-eloquent-js/) - Marijn Haverbeke, Rabah Boudia (gitbook) 
 
 
+#### Next.js
+
+* [سلسلة تعلم Next.js بالعربية](https://blog.abdelhadi.org/learn-nextjs-in-arabic/) - Flavio Copes, عبدالهادي الأندلسي
```
Comment (davorpa) [inline books/free-programming-books-ar.md:64]:
```suggestion
* [سلسلة تعلم Next.js بالعربية](https://blog.abdelhadi.org/learn-nextjs-in-arabic/) - Flavio Copes, عبدالهادي الأندلسي
* [Eloquent Javascript (Second Edition)&rlm;](https://rabahboudia.gitbooks.io/arabic-eloquent-js/) - Marijn Haverbeke, Rabah Boudia (gitbook)
```

**Review by davorpa — CHANGES_REQUESTED** [review]
> Being that it is a single resource and Next.js is a framework of JavaScript I propose to avoid to create a new category and to insert it on JavaScript

+1

File: books/free-programming-books-ar.md (hunk 1 above)
Comment (AhmedElTabarani) [reply books/free-programming-books-ar.md:11]: I can't commit this suggestion, github disable the buttons

**Review by davorpa — APPROVED** [review]

**Pull request merged**